README.md

```diff
@@ -1,41 +1,80 @@
 
-The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker/Singularity containers making installation trivial and results highly reproducible. The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per process which makes it much easier to maintain and update software dependencies. Where possible, these processes have been submitted to and installed from [nf-core/modules](https://github.com/nf-core/modules) in order to make them available to all nf-core pipelines, and to everyone within the Nextflow community!
+## Installation locally
 
-<!-- TODO: Add full-sized test dataset and amend the paragraph below if applicable -->
+You'll need to have [cookiecutter](https://cookiecutter.readthedocs.io/) installed.
 
-## Pipeline summary
+```{bash}
+pip install cookiecutter
+```
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps in the pipeline -->
+## Usage
 
-## Quick start
+Run `cookiecutter gh:Arcadia-Science/nextflow-template` and then answer the prompts. Here's an example run:
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps to get the pipeline up and running -->
+```{bash}
+$ cookiecutter gh:simonw/click-app
+```
 
-## Full documentation
+It is strongly recommended to accept the suggested value for "hyphenated" and "underscored" by hitting enter on those prompts.
 
-<!-- TODO: Fill in this section with how to fully use the pipeline, what the inputs are and what the outputs look like. If this section ends up being super long, feel free to create a new docs/ directory and add details there. -->
+Having created the new structure from the template, here's how to start working on the tool.
 
-## Contributions and support
+If your tool is called `my-new-tool`, you can start working on it like so:
 
-<!-- TODO: Add CONTRIBUTING.MD that is specific to Arcadia Science -->
+    cd my-new-tool
+    # Create and activate a virtual environment:
+    python3 -m venv venv
+    source venv/bin/activate
+    # Install dependencies so you can edit the project:
+    pip install -e '.[test]'
+    # With zsh you have to run this again for some reason:
+    source venv/bin/activate
+    # Confirm your tool can be run from the command-line
+    my-new-tool --version
 
-## Citations
+You should see the following:
 
-<!-- TODO: Add bibliography of tools and data used in your pipeline -->
+    my-new-tool, version 0.1
 
-An extensive list of references for the tools used by the pipeline can be found in the [`CITATIONS.md`](CITATIONS.md) file.
+You can run the default test for your tool like so:
+
+    pytest
+
+This will execute the test in `tests/test_my_new_tool.py`.
+
+Now you can open the `my_new_tool/cli.py` file and start adding Click [commands and groups](https://click.palletsprojects.com/en/7.x/commands/).
```

Comment:
This feels like an intimidating statement...why do I need click to use this template and what is it's function within this template?